packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/container.tsx

```diff
@@ -93,24 +93,15 @@ const CompleteResetPasswordContainer = ({
     'Don’t forget to generate a new account recovery key from your Mozilla account settings to prevent future sign-in issues.'
   );
 
-  const signInNavigationDestination = (email: string) =>
-    location.search?.includes('email')
-      ? // if the session is not verified (e.g., 2FA verification is required), navigate to the sign-in page
-        '/signin'
-      : // if user started directly from the reset password page without passing GO (index),
-        // the email needs to be added to query params to avoid redirecting to the index page
-        `/signin?email=${encodeURIComponent(email)}`;
-
   const handleNavigationWithRecoveryKey = (
     state: Record<string, any>,
     hasVerifiedSession: boolean
   ) => {
     if (!hasVerifiedSession) {
-      const destination = signInNavigationDestination(state.email);
-
-      return navigateWithQuery(destination, {
+      return navigateWithQuery('/signin', {
```

Comment:
Self-review: look into cleaning up unnecessary query params in useNavigateWithQuery hook

Reply:
Follow-up: https://mozilla-hub.atlassian.net/browse/FXA-10567